hlib/python/hlib/frontend/__init__.py

```diff
@@ -0,0 +1,3 @@
+from . import relay_parser
+from . import test
```

Comment:
We should rename "test" to "tests". And "tests" should also not be included in ``__init__.py``.